Tools/CISupport/delete-stale-build-files

```diff
@@ -86,6 +91,21 @@ def webkitBuildDirectory(platform, fullPlatform, configuration):
     return subprocess.Popen(['perl', os.path.join(os.path.dirname(__file__), "..", "Scripts", "webkit-build-directory"),
         "--" + platform, "--" + configuration, '--top-level'], stdout=subprocess.PIPE).communicate()[0].strip()
 
+def deleteWindowsStaleFiles():
+    directoriesToDelete = ['/tmp',
+                           '/cygdrive/c/Program Files (x86)/Windows Kits/10/Debuggers/x64/sym/WebKit.pdb',
+                           '/cygdrive/c/Program Files (x86)/Windows Kits/10/Debuggers/x64/sym/WTF.pdb',
```

Comment:
I think it's possible to determine the `Debuggers` folder with environment variable. It will make this good resilient to updates.

Reply:
Any reason not to just delete all the Debuggers?

Reply:
No reason, I wasn't sure if it's fine to delete it totally (e.g.: not sure if those files are used as a cache to speed up subsequent builds). Those were the directories which were large (in GBs, rest all were smaller). e.g.:

buildbot@ews210 /cygdrive/c/Program Files (x86)/Windows Kits/10/Debuggers/x64/sym
$ du -hs * | grep G
21G     DumpRenderTreeLib.pdb
21G     JavaScriptCore.pdb
4.1G    WTF.pdb
133G    WebKit.pdb